netbox_onboarding/tests/test_onboarding_addons_ios.py

```diff
@@ -0,0 +1,119 @@
+"""Unit tests for netbox_onboarding.naplam_addons.ios module and its classes.
+
+(c) 2020 Network To Code
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+  http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+# pylint disable=protected-access
+# Python First Party Imports
+import os
+
+# Python Third Party Imports
+from django.test import TestCase
+from napalm import get_network_driver
+
+# NetBox Imports
+from dcim.models import Platform
+
+# Application Imports
+from netbox_onboarding.onboard import NetdevKeeper, OnboardException
+
+# Onboard class as the name of the platformExtension to allow for the onboarding of several platforms
+from netbox_onboarding.onboarding_extensions.ios import OnboardingDriverExtensions as IosExtension
+
+# Load utilities
+from .utilities import load_test_output
+
+
+class IosExtensionTestCase(TestCase):
+    """Test the IOS NapalmDeviceExtensions."""
+
+    def test_parse_output(self):
+        """Method to test parsing of output through NTC Templates parser"""
+        test_output_dir = "/source/netbox_onboarding/tests/fixtures"
```

Comment:
This shouldn't be a hard-coded absolute path - use `os.path` relative to `__file__` instead - something like `os.path.join(os.path.dirname(__file__), 'fixtures')` I think.